ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -369,10 +367,11 @@ def yield_data_models(self, dbt_files: DbtFiles) -> Iterable[DataModelLink]:
                         ),
                         datamodel=DataModel(
                             modelType=ModelType.DBT,
-                            description=manifest_node.get("description")
-                            if manifest_node.get("description")
+                            description=manifest_node.get("description"),
```

Comment:
what will happen if description is not there

Reply:
@ulixius9 should we do this ```manifest_node.get("description", {})```

Reply:
@harshach description is of type [markdown](https://github.com/open-metadata/OpenMetadata/blob/8b45b639fd59abaf95baf55c3a46304bd10859eb/openmetadata-spec/src/main/resources/json/schema/entity/data/table.json#L767) and is an optional field so it's okay for description to be `None`

Reply:
@ulixius9 talking about manifest_node.get("description") , description is an optional element from dbt manifest schema. Which means we may not have description as the key in manifest file which will throw up an error from get method saying key not found

Reply:
@harshach  Yes the `description` is an optional field in manifest and `manifest_node.get("description")` will return `None` if the key `description` is not found 


here is an example 
![image](https://user-images.githubusercontent.com/39544459/215250239-921be710-de51-4d40-acb4-20696b73bd18.png)